interfaces/Portalicious/src/app/services/log.service.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable } from '@angular/core';
+import { DebugPlugin } from '@microsoft/applicationinsights-debugplugin-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { environment } from 'src/environments/environment';
+
+// All properties sorted alphabetically.
+export enum LogEvent {
```

Comment:
Maybe we wanna add this? https://github.com/infctr/eslint-plugin-typescript-sort-keys/blob/master/docs/rules/string-enum.md

Reply:
Ah, yes. If we limit it to TS-only, then there's not really any risk and things can be autofixed...

The only issues in the past was with "hand-editted JSON-files", for those the sorting made it "less understandable"..

Reply:
@aberonni I've found the perfectly named plugin for that:
https://eslint-plugin-perfectionist.azat.io/rules/sort-enums

I'll add in a separate PR. (together with some other ESLint-things)